File: src/main/java/framework/frameworkUtilies/PropertiesConfigurator.java
package framework.frameworkUtilies;

import java.io.FileInputStream;
import java.io.InputStream;
import java.util.Properties;

import org.apache.log4j.Logger;

import framework.startup.Startup;

public class PropertiesConfigurator
{

	static Properties		prop	= null;
	public static Logger	logger	= Logger.getLogger(PropertiesConfigurator.class);

	public static Properties loadProperties() throws Exception
	{

		InputStream input = new FileInputStream("./Resources/config.properties");
		prop = new Properties();
		prop.load(input);
		for (Object key : prop.keySet())
		{
			Startup.configMap.put(key.toString(), prop.get(key).toString());
		}
		logger.info("loadProperties  :  " + Startup.configMap);

		return prop;
	}

	public static Properties loadSystemEnvVariable() throws Exception
	{
		for (String key : System.getenv().keySet())
		{
			Startup.configMap.put(key, System.getenv(key));
		}
		logger.info("loadSystemEnvVariable  :  " + Startup.configMap);
		return null;
	}

}

File: src/Framework/PropertiesConfigurator/PropertiesConfigurator.ts
import { Startup } from "../FrameworkUtilities/Startup/Startup";
import { prop } from "../FrameworkUtilities/config";
import { logger } from "../FrameworkUtilities/Logger/logger";

export function parseJson() {
  for (const [key, value] of Object.entries(prop)) {
    Startup.configmap.set(key, value);
    //logger.info(Startup.testData);
  }

  // for (const [key, value] of Startup.configmap.entries()) {
  //   logger.info(`key --> ${key} || value --> ${value}`);
  // }

}


